_data/navigation.yml

```diff
@@ -1,3 +1,15 @@
+- name: Documentation
+  links:
+  - name: Quickstart
+    link: /quickstart/
+  - name: API reference
+    link: /api_reference/
+  - name: Use Cases
+    link: /usecases/
+
+- name: Access the tool
+  link: /access/
```

Comment:
/access/ is converted to https://solarperformanceinsight.org/access/ which does not exist. I think you want https://app.solarperformanceinsight.org/